simtools/util/names.py

```diff
@@ -197,6 +197,21 @@ def validateTelescopeName(name):
         Validated name.
     '''
     telSite, telClass, telType = splitTelescopeName(name)
+    telSite = validateSiteName(telSite)
+    telClass = validateName(telClass, allTelescopeClassNames)
+    if 'flashcam' in telType:
+        telType = telType.replace('flashcam', 'FlashCam')
+    if 'nectarcam' in telType:
+        telType = telType.replace('nectarcam', 'NectarCam')
+    if '1m' in telType:
+        telType = telType.replace('1m', '1M')
+    if 'gct' in telType:
+        telType = telType.replace('gct', 'GCT')
+    if 'astri' in telType:
+        telType = telType.replace('astri', 'ASTRI')
+    if '-d' in '-' + telType:
+        telType = telType.replace('d', 'D')
```

Comment:
This one is a little dangerous, as it will change any "d" into a "D". Perhaps worth changing to

`replace('-d', '-D')`

Though I agree that if names become complicated, we will have to come up with smarter solutions (or give up on this generally).

Reply:
That was my first attempt, but it happens that the telType does not contains the '-' anymore. That is why I searched for '-d' in '-' + telType.
If the name is *-d, it will work fine and replace by *-D. If it is like '-bla-d', it will also work fine. The only bad case is something like *-d-dla, which will become *-D-Dla.

I think it's fine for now, since we have full control over all the telescope names, which are not many. In the future we can redo this in a better way.

Reply:
OK, sounds good.